app/models/school.rb

```diff
@@ -144,6 +148,19 @@ def induction_tutor
     induction_coordinators.first
   end
 
+  def slug_candidates
+    [
+      :name,
+      %i[name urn],
+    ]
+  end
+
+# This temporary code is temporarily commented out during development
+# # This is temporary so that nothing breaks until the slugs are generated
+# def to_param
+#   id
+# end
```

Comment:
mhmmmm.

Development is over, do we uncomment or remove it?

Reply:
this will be uncommented (and half the code removed) before merge but after review

Reply:
Wait, I haven't read it properly.

Will we need another PR removing this method after we have generated slugs in all environments?

Reply:
yep!